tests/integration/security/rbac/v1/group/group_test.go

```diff
@@ -15,19 +15,18 @@
 package group
 
 import (
-	"fmt"
 	"testing"
 	"time"
 
-	"istio.io/istio/pilot/pkg/model"
+	util2 "istio.io/istio/tests/integration/security/util"
```

Comment:
`util2` is confusing with `util`, please use a different name, here and everywhere